cypress/e2e/tests/pages/about.spec.ts

```diff
@@ -0,0 +1,165 @@
+import * as path from 'path';
+
+const aboutPage = new AboutPagePo();
+const downloadsFolder = Cypress.config('downloadsFolder');
+
+describe('About Page', { testIsolation: 'off', tags: ['@adminUser', '@standardUser'] }, () => {
+  before(() => {
+    cy.login();
+  });
+
+  it('can navigate to About page', () => {
+    HomePagePo.goToAndWaitForGet();
+
+    const burgerMenu = new BurgerMenuPo();
+
+    BurgerMenuPo.toggle();
+    burgerMenu.about().click();
+    aboutPage.waitForPage();
+  });
+
+  it('can navigate to Diagnostics page', () => {
+    aboutPage.goTo();
+    aboutPage.waitForPage();
+    aboutPage.diagnosticsBtn().click();
+
+    const diagnosticsPo = new DiagnosticsPagePo();
+
+    diagnosticsPo.waitForPage();
+  });
+
+  it('can View release notes', () => {
+    aboutPage.goTo();
+    aboutPage.clickVersionLink('View release notes');
+    cy.origin('https://github.com/rancher/rancher', () => {
+      cy.url().should('include', 'https://github.com/rancher/rancher/releases/tag/');
+    });
+  });
+
+  describe('Versions', () => {
+    beforeEach(() => {
+      aboutPage.goTo();
+    });
+
+    it('can see rancher version', () => {
+      // Check Rancher version
+      cy.getUserPreferences().then((resp: Cypress.Response<any>) => {
+        const rancherVersion = JSON.parse(resp.body.data[0]['data']['seen-whatsnew']);
+
+        cy.contains(rancherVersion).should('be.visible');
+      });
+    });
+
+    it('can navigate to /rancher/rancher', () => {
+      aboutPage.clickVersionLink('Rancher');
+      cy.origin('https://github.com/rancher/rancher', () => {
+        cy.url().should('include', 'https://github.com/rancher/rancher');
+      });
+    });
+
+    it('can navigate to /rancher/dashboard', () => {
+      aboutPage.clickVersionLink('Dashboard');
+      cy.origin('https://github.com/rancher/dashboard', () => {
+        cy.url().should('include', 'https://github.com/rancher/dashboard');
+      });
+    });
+
+    it('can navigate to /rancher/helm', () => {
+      aboutPage.clickVersionLink('Helm');
+      cy.origin('https://github.com/rancher/helm', () => {
+        cy.url().should('include', 'https://github.com/rancher/helm');
+      });
+    });
+
+    it('can navigate to /rancher/machine', () => {
+      aboutPage.clickVersionLink('Machine');
+      cy.origin('https://github.com/rancher/machine', () => {
+        cy.url().should('include', 'https://github.com/rancher/machine');
+      });
+    });
+  });
+
+  describe('Image List', () => {
+    beforeEach(() => {
+      aboutPage.goTo();
+    });
+
+    it('can download Linux Image List', () => {
+      // Download txt and verify file exists
+      const downloadedFilename = path.join(downloadsFolder, 'rancher-linux-images.txt');
+
+      aboutPage.getLinuxDownloadLink().click();
+
+      cy.getUserPreferences().then((resp: Cypress.Response<any>) => {
+        const rancherVersion = JSON.parse(resp.body.data[0]['data']['seen-whatsnew']);
```

Comment:
The rancher version can be fetched by making a request to `/rancherversion`

Reply:
when fetching the rancher version via `/rancherversion`, it returns a partial git hash and not the full hash like `v2.7-{full hash}-head` which is why i went the `/v1/userpreferences` route.

Reply:
good call! i'll give it a shot

Reply:
@richard-cox should I use cy.getRancherResource() instead of creating a new custom command?

`cy.getRancherResource('v1', 'management.cattle.io.settings', 'server-version')`